tests/queries/0_stateless/02698_marked_dropped_tables.sql

```diff
@@ -4,8 +4,11 @@ SET database_atomic_wait_for_drop_and_detach_synchronously = 0;
 DROP TABLE IF EXISTS 25400_dropped_tables;
 
 CREATE TABLE 25400_dropped_tables (id Int32) Engine=MergeTree() ORDER BY id;
+INSERT INTO 25400_dropped_tables VALUES (1),(2);
+INSERT INTO 25400_dropped_tables VALUES (3),(4);
 DROP TABLE 25400_dropped_tables;
 
 SELECT table, engine FROM system.dropped_tables WHERE database = currentDatabase() LIMIT 1;
 DESCRIBE TABLE system.dropped_tables;
- 
+SELECT database, table, name FROM system.dropped_tables_parts WHERE database = currentDatabase() and table = '25400_dropped_tables';
```

Comment:
Let's also add a test that runs `select * from system.dropped_tables_parts format Null` and works in parallel with other tests

Reply:
`... format Null` will not test the issue with Ordinary - I added a test which will...

Reply:
Yes, but the purpose of this query is to find race conditions, not issues with Ordinary